configure/migrator/migrator.Job.yaml

```diff
@@ -0,0 +1,184 @@
+# These manifests are provided in the event that someone may wish to execute
+# database migrations outside of a code deployment.
+#
+# For more information, please see: https://docs.sourcegraph.com/admin/install/kubernetes/operations#database-migrations
+
+# ---
+# apiVersion: batch/v1
+# kind: Job
+# metadata:
+#   name: migrator
+#   annotations:
+#     description: Database migrator for Postgres databases
+#   labels:
+#     app.component.kubernetes/component: migrator
+#     deploy: sourcegraph
+# spec:
+#   template:
+#     spec:
+#       containers:
+#       - name: migrator
+#         image: "sourcegraph/migrator:3.36.0"
```

Comment:
Can you update this to follow the same pattern as our other deployments - something similar to:
`image: index.docker.io/sourcegraph/frontend:insiders@sha256:f6b962027b6cdb4a1cdea2573d9dfd89f353e2efb5ac2c485bf086bfa01e7a93`

I can't say for sure but I think we rely on that pattern to do some checks (https://sourcegraph.com/github.com/sourcegraph/deploy-sourcegraph/-/blob/tools/check-image-names/check-image-names.go?L17). Super nitpicky, sorry!

Reply:
I just pushed the change with the digest pattern and the correct 3.36.1 image

https://hub.docker.com/layers/sourcegraph/migrator/3.36.1/images/sha256-54a1b8db471b0d13d02bf730551cb748d8362cf3bb35c09f20aa02b8c918c56d?context=explore